impress-org/givewp-next-gen
# Feature: Form Builder Onboarding (Tour)

## Description



This PR adds a form builder onboarding tour using ShepherdJS.

## Visuals



![image](https://user-images.githubusercontent.com/10858303/228354071-db707d4d-3d71-49bb-9efa-dc5e03862834.png)

![Peek 2023-03-22 16-58](https://user-images.githubusercontent.com/10858303/227036618-2114672b-aba9-4f24-bde9-afbb5a6676c5.gif)

![Peek 2023-03-28 13-48](https://user-images.githubusercontent.com/10858303/228324868-309ad17b-bc35-4072-b95b-5d727841a7b1.gif)


## Testing Instructions



Create a new next gen donation form and step through the tour. Once complete, the tour will no longer auto-start on page load. To re-start the tour delete the local storage key `givewp-next-gen-onboarding-complete`.
